packages/synapse-react-client/src/components/TableColumnSchemaEditor/ColumnModelFormFields/DefaultValueField.tsx

```diff
@@ -119,10 +115,8 @@ function DefaultValueListField(props: DefaultValueFieldProps<string>) {
     <MultiValueField
       value={value}
       onChange={arr => {
-        if (arr == null) {
-          onChange(undefined)
-        }
-        onChange(JSON.stringify(arr))
+        // Convert null to undefined
+        onChange(arr || undefined)
```

Comment:
This may return ISO strings when the backend gave us numbers, but that's OK! The backend is permissive in terms of what it accepts